pkg/operator/controller/controller.go

```diff
@@ -71,19 +69,12 @@ const (
 
 var (
 	log = logf.Log.WithName("cdi-operator")
-
-	readyGauge = prometheus.NewGauge(
-		prometheus.GaugeOpts{
-			Name: monitoring.MetricOptsList[monitoring.ReadyGauge].Name,
-			Help: monitoring.MetricOptsList[monitoring.ReadyGauge].Help,
-		})
 )
 
 func init() {
-	metrics.Registry = prometheus.NewRegistry()
-	metrics.Registry.MustRegister(readyGauge)
-	// 0 is our 'something bad is going on' value for alert to start firing, so can't default to that
-	readyGauge.Set(-1)
+	// Setup metrics for our various controllers
+	metrics.SetupMetrics()
```

Comment:
this should be in `cmd/cdi-operator/operator.go` as you did in `cmd/cdi-controller/controller.go` on this PR
and therefore the package name in `pkg/monitoring` should be `pkg/monitoring/cdi-operator/metrics/...`